apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/IterableLikeToContainCheckerSamples.kt

```diff
@@ -27,4 +27,28 @@ class IterableLikeToContainCheckerSamples {
             }
         }
     }
+
+    @Test
+    fun notOrAtMost() {
+        expect(listOf("A", "A", "B", "C")).toContain.inAnyOrder.notOrAtMost(2).entry {
+            toEqual("A")
+        }
+
+        expect(listOf(1, 2, 3)).toContain.inAnyOrder.notOrAtMost(2).entry {
+            toBeGreaterThan(1)
```

Comment:
how about you change this to 4 and add a comment why this expectation holds:
```suggestion
            // none fulfils the expectation which is fine because we use notOrAtMost
            // use atMost if you want that at least one element matches
            toBeGreaterThan(4) 
```